src/components/StackNavigation/StackNavigation.tsx

```diff
@@ -10,8 +10,8 @@ import {StackNavigationDots} from "./Dots/StackNavigationDots";
 interface StackNavigationProps {
   stacks: Note[];
   currentStack: string;
-  prevColumnStack?: string;
-  nextColumnStack?: string;
```

Comment:
Why?

Reply:
Wouldn't `null` be better than `undefined`?

Reply:
can't remember, changed it back